app/javascript/components/pages/Modal.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+// import { getConstructor } from "react_ujs";
```

Comment:
ahhhh!!!

Reply:
In the voice from "it's a wonderful life": every time commented code goes in, an angel loses its wings.

Reply:
dammit saw it too late